Resources/Maps/_NF/Shuttles/mayflower.yml

```diff
@@ -4011,12 +4059,72 @@ entities:
       pos: 6.5,-16.5
       parent: 1
       type: Transform
+  - uid: 341
+    components:
+    - pos: 3.5,-17.5
+      parent: 1
+      type: Transform
   - uid: 342
     components:
     - rot: -1.5707963267948966 rad
       pos: -8.5,-16.5
       parent: 1
       type: Transform
+  - uid: 351
+    components:
+    - pos: 5.5,-16.5
+      parent: 1
+      type: Transform
+  - uid: 354
+    components:
+    - pos: 4.5,-16.5
+      parent: 1
+      type: Transform
+  - uid: 355
+    components:
+    - pos: 2.5,-15.5
+      parent: 1
+      type: Transform
+  - uid: 360
+    components:
+    - pos: 3.5,-16.5
+      parent: 1
+      type: Transform
+  - uid: 363
+    components:
+    - pos: 4.5,-17.5
+      parent: 1
+      type: Transform
+  - uid: 370
+    components:
+    - pos: 4.5,-15.5
+      parent: 1
+      type: Transform
+  - uid: 381
+    components:
+    - pos: -6.5,-16.5
```

Comment:
Please do not stack more spider webs on top of spider webs

Reply:
This has been a thing since the beginning of the ship.

Reply:
This isnt a good thing

Reply:
Just because it wasn't noticed in the past does not mean it wasn't a bad thing before, it was a bad thing that went un-noticed.  Please listen to dvir.